config/default.yaml

```diff
@@ -75,6 +75,7 @@ storage:
 
 log:
   level: 'info' # debug/info/warning/error
+  rotation: true # Enabled by default, if disabled make sure that 'storage.logs' is pointing to a folder handled by logrotate
```

Comment:
Prefer:

```
  rotation:
    enabled: true
```

So we can add other parameters in the future (file max size etc)

Please update https://github.com/Chocobozzz/PeerTube/blob/develop/config/production.yaml.example too

Reply:
Thank you, I did it ! By doing this, it will be easier to complete [this request](https://github.com/Chocobozzz/PeerTube/issues/1775#issuecomment-498587347).